ci/build.sh

```diff
@@ -4,12 +4,55 @@ set -e -u -x -o pipefail
 
 cd "$(dirname "$0")/.."
 
+if [[ -z ${BUILDKITE+x} ]]; then
+  echo "This script is intended for internal Improbable use."
```

Comment:
Maybe: This script is intended to run in CI only?

Reply:
Sure, changed.